package.json

```diff
@@ -17,7 +17,7 @@
   "author": "Henrik Joreteg <henrik@andyet.net>",
   "license": "MIT",
   "dependencies": {
-    "getusermedia": "^2.0.0"
+    "webrtc-adapter": "^3.1.6"
```

Comment:
We've talked in other modules about starting to make adapter.js a peerDependency instead of a direct dependency. Want to do that in this version?